README.md

```diff
@@ -201,6 +203,16 @@ The reponse also includes the counts of simulator and quantum computer as well a
 
 `GET /qiskit-service/api/v1.0/analysis`
 
+Access the analysis of all benchmarks on a specific Quantum Computer in the database that successfully returned a result with
+
+`GET /qiskit-service/api/v1.0/analysis/<qpu_name> `
+
+## wd Request
+Request the wd-value of a specific Quantum Computer based on the clifford gate circuit data in your database
```

Comment:
also describe how many successful /randomize invocations are required to get a result

Reply:
also add an interpretation of the result value, so what das data count mean?